cmake/SetupCompilerOptions.cmake

```diff
@@ -325,7 +325,9 @@ blt_append_custom_compiler_flag(
 if ( COMPILER_FAMILY_IS_MSVC AND NOT BUILD_SHARED_LIBS )
     foreach(flag_var
             CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE
-            CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO)
+            CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO
```

Comment:
Would it make sense to refactor this into two loops?
E.g.

```cmake
foreach(lang C CXX)
   foreach(build FLAGS FLAGS_DEBUG ....)
        ... ${CMAKE_${lang}_${build}}  ...
   endforeach()
endforeach()
```
That would make it easier to read and to extent to other languages (e.g. Fortran, if necessary) and/or build types (if necessary)

Reply:
I'm attempting to fix it but messing something up :(